animations/lib/src/misc/focus_image.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:flutter/material.dart';
+
+class FocusImageDemo extends StatelessWidget {
+  static String routeName = '/misc/focus_image';
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text('Focus Image')),
+      body: Grid(),
+    );
+  }
+}
+
+class Grid extends StatelessWidget {
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: GridView.builder(
+        itemCount: 40,
+        gridDelegate:
+        SliverGridDelegateWithFixedCrossAxisCount(crossAxisCount: 4),
+        itemBuilder: (context, index) {
+          return (index >= 20)
+              ? SmallCard('assets/eat_cape_town_sm.jpg')
+              : SmallCard('assets/eat_new_orleans_sm.jpg');
+        },
+      ),
+    );
+  }
+}
+
+Route _expandToPageRoute(BuildContext parentContext, String image) {
```

Comment:
nit: Consider a name that clearly indicates the function is creating a route object (e.g. "_createExpandToPageRoute").